src/fb.md

```diff
@@ -1,40 +1,52 @@
 
-<!-- (on a side note: x2y2 means that you're more sensitive to the L/R sides as their colors are fixed.  ) -->
+It is recommended to watch videos on blockbuilding strategies and have a basic understanding before you proceed to example solves.
 
-Regarding full CN: it is a plus, but the benefits have not been proven as there has yet to be a world class full CN Rouxer. Some speculate the benefits would be minimal because there is limited time during inspection to analyse all 24 FB options. However, a counterpoint would be that for any first square, you have two choices of last pair to extend to a FB, making the last pair case much better on average. Therefore, once you find a good first square, you can usually settle on it and make better use of your inspection time by looking further into the solve. 
+Overall strategy:
+- [Finding an Efficient FB in Inspection](https://www.youtube.com/watch?v=0Cq3YDud1dA)
+- [Louis' First Block Tutorial](https://www.youtube.com/playlist?list=PLAmLIbUNCUR_e9lmyvcjAb18qlnJ72hGv)
+- [Iuri's Line + Line first block](https://www.youtube.com/watch?v=i9zxR5mkgQs)
 
-TL;DR: CN Roux is worth exploring, but don't force yourself.
+Beginner-friendly Example Solves:
+- Kavin's Example Solves.
+  - [Pair + Pair Example Solves](https://www.youtube.com/watch?v=sRTVptb2QrY)
+  - [Square + Pair Example Solves](https://www.youtube.com/watch?v=YyY_okZ5Fj0)
+  - These illustrate pure `square + pair` FB building -- they don't involve advanced stuff like influencing.
 
-If you are interested in the average movecount comparison of FB or FB + DR under different orientations, use the following resource:
+### Relationship to Planning
+Before we go on, note that `planning = solution + tracking`, with solution playing perhaps a more important role. For now stay stuned on getting you solutions right, and we'll cover tracking in the last section.
 
-- [Movecount statistics](https://docs.google.com/spreadsheets/d/1EectP3O_qwQp_2WohbDrlohxhs2ee17sjFfxZr_-D1g/edit#gid=0) - Notice how x2y boosts chance of easy first square compared to y from 60% to 80%.
+### Step II: Improve Your Solutions
 
-## Blockbuilding Strategies
+Next, you want to improve your solutions. There are several aspects you can work on, in no particular order. Not only are easier solutions faster to execute, but the ability to come up with them so will also help bring down your inspection time.
+g.
 
-There are broadly speaking two families of FB building strategy: square + pair and line + line. See resources below:
+- Influencing
+    - `square + pair` and `line1 + line2` are both two-stage solutions, and the point is we want to find alternative ways to solve the first stage i.e. `square`/`line1` so we get a better second stage case i.e. `pair`/`line2`.
```

Comment:
what is line1/line2 - is that like the E line and the D line?

Reply:
maybe footnotes for these would be nice to have

Reply:
good point